src/theory/theory_model.h

```diff
@@ -318,6 +316,11 @@ class TheoryModel
   RepSet* getRepSetPtr() { return &d_rep_set; }
 
   //---------------------------- model cores
+  /**
+   * Is using model core (has a model core already been computed for this
```

Comment:
That's a bit of a weird way to document this IMO. There's no added value to just repeating the name of the function. Maybe be more explicit and write
```suggestion
   * True if a model core has already been computed for this for this model.
```